core/coreobjects/include/coreobjects/callable_info.h

```diff
@@ -68,7 +76,8 @@ DECLARE_OPENDAQ_INTERFACE(ICallableInfo, IBaseObject)
 OPENDAQ_DECLARE_CLASS_FACTORY(
     LIBRARY_FACTORY, CallableInfo,
     IList*, argumentInfo,
-    CoreType, returnType
+    CoreType, returnType,
```

Comment:
I would consider making a new factory here `CallableInfoWithConst` to preserve backwards compatibility. It's not 100% mandatory, as I don't think anyone is creating openDAQ properties in any bindings. 

At the very least the Integration notes of the changelog should notify that a factory signature got changed and needs to be fixed in all modules that use the C factory.

Reply:
I have added a note to changelog file.